contrib/roles/linux/kubernetes/tasks/download_k8s_binaries.yml

```diff
@@ -44,6 +44,8 @@
         remote_src: yes
 
     - name: Kubernetes bins | download kubernetes binaries
+      environment:
+        http_proxy: "{{KUBERNETES_DOWNLOAD_PREFIX}}"
```

Comment:
Is this working? You're trying to download the binaries using the proxy "https://dl.k8s.io"?

Reply:
nope, i messed up the env var, sorry for that, i fixed it